config/resources/domain.lisp

```diff
@@ -158,14 +158,6 @@
                               :as "publisher"))
   :has-many `((editor-document :via ,(s-prefix "pav:hasVersion")
                                :as "revisions")
-              (versioned-agenda :via ,(s-prefix "ext:hasVersionedAgenda")
-                                :as "versioned-agendas")
-              (versioned-behandeling :via ,(s-prefix "ext:hasVersionedBehandeling")
```

Comment:
Why are you removing the reference to the versioned resources from the document container? Not saying is bad, but don't know why it would be good

Reply:
because document containers no longer have references to them (no relationships to be found in the database), additionally we also do not use them in the frontend.